app/src/main/java/com/example/taskyy/ui/LoginActivity.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.taskyy.ui
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+
+class LoginActivity : ComponentActivity() {
```

Comment:
If you're using Compose, you're totally fine with a single activity, so this could just be a generic `MainActivity`